backend/Dockerfile.prod

```diff
@@ -1,9 +1,20 @@
-FROM gradle:6.6.1-jdk11-hotspot AS build
-COPY --chown=gradle:gradle . /home/gradle/src
-WORKDIR /home/gradle/src
-RUN gradle build --no-daemon -x test
+FROM gradle:6.7.1-jdk11-hotspot AS build
```

Comment:
The reason this was 6.6.1 is because we are currently using 6.6.1 for development and builds—if there's a compelling reason to upgrade to 6.7.1, we need to update the the wrapper configuration and the other Dockerfile(s?).

Reply:
Heh, looks like the other Dockerfile is already on the "wrong" version (didn't spot that before). So probably we should just update the wrapper and move everybody up to 6.7.1?